server/vulnerabilities/cpe.go

```diff
@@ -44,6 +45,8 @@ func GetLatestNVDRelease(client *http.Client) (*NVDRelease, error) {
 
 	cpeURL := ""
 
+	// TODO: get not draft release
```

Comment:
Ah yes, we still need to 1) check the latest not draft release (just an improvement), and 2) move the nvd repo to `fleetdm` and generate the db on a schedule.

Reply:
I just created https://github.com/fleetdm/nvd, please use that.

Reply:
Hm, I think I might need to be added as a contributor? 

<img width="1200" alt="Screen Shot 2021-08-03 at 18 06 03" src="https://user-images.githubusercontent.com/167183/128086083-fe0d0407-781e-42c0-9023-c9d9f8d10796.png">

First time I see that view of a repo.

Reply:
Hopefully when you check again it will give you the info you need to push to it.